xbmc/network/EventServer.cpp

```diff
@@ -186,7 +182,7 @@ void CEventServer::Run()
     CLog::Log(LOGERROR, "ES: Invalid port range specified %d, defaulting to 10", port_range);
     port_range = 10;
   }
-  if (!m_pSocket->Bind(any_addr, m_iPort, port_range))
+  if (!m_pSocket->Bind(!CSettings::GetInstance().GetBool(CSettings::SETTING_SERVICES_ESALLINTERFACES), m_iPort, port_range))
```

Comment:
By default ES is localhost only, with a settings to open to all but is needed by remote :)

Maybe @Paxxi does not know but I'm the author of Yatse and currently 1 million monthly users do rely on ES accessible from LAN ;) Kore have added it recently also with limited things.

ES on local host is used too by some tools like on mac for some hardware remotes.

So nothing have to be changed on settings and global usage, it's just about adding support for IPV6 to something that already works and fit needs.

For information I fixed ES implementation recently, before there was a huge security problem as by default ES was on on all interfaces.

I have long history with ES ;)